tasks.c

```diff
@@ -2216,15 +2234,15 @@ static void prvAddNewTaskToReadyList( TCB_t * pxNewTCB )
         {
             pxTCB = prvGetTCBFromHandle( xTask );
 
-            pxTCB->uxCoreExclude = uxCoreExclude;
+            pxTCB->uxCoreAffinityMask = uxCoreAffinityMask;
 
             if( xSchedulerRunning != pdFALSE )
             {
                 if( taskTASK_IS_RUNNING( pxTCB->xTaskRunState ) )
                 {
                     xCoreID = ( BaseType_t ) pxTCB->xTaskRunState;
 
-                    if( ( uxCoreExclude & ( 1 << xCoreID ) ) != 0 )
+                    if( ( uxCoreAffinityMask & ( 1 << xCoreID ) ) != 0 )
```

Comment:
Suggestion: This bitmask check could be done in a macro for consistency throughout the file.